plugin/pkg/replacer/replacer.go

```diff
@@ -31,7 +33,7 @@ type replacer struct {
 // values into the replacer. rr may be nil if it is not
 // available. emptyValue should be the string that is used
 // in place of empty string (can still be empty string).
-func New(r *dns.Msg, rr *dnstest.Recorder, emptyValue string) Replacer {
+func New(ctx context.Context, r *dns.Msg, rr *dnstest.Recorder, emptyValue string) Replacer {
```

Comment:
You'll have to update the tests to factor in the context.

Reply:
Yes, you are right. I fixed it in the tests.